Content.Server/Bible/BibleSystem.cs

```diff
@@ -81,15 +79,43 @@ private void OnAfterInteract(EntityUid uid, BibleComponent component, AfterInter
                 }
             }
 
-            var othersMessage = Loc.GetString("bible-heal-success-others", ("user", args.User),("target", args.Target),("bible", uid));
+            var othersMessage = Loc.GetString(component.LocPrefix + "-heal-success-others", ("user", args.User),("target", args.Target),("bible", uid));
             _popupSystem.PopupEntity(othersMessage, args.User, Filter.Pvs(args.User).RemoveWhereAttachedEntity(puid => puid == args.User));
 
-            var selfMessage = Loc.GetString("bible-heal-success-self", ("target", args.Target),("bible", uid));
+            var selfMessage = Loc.GetString(component.LocPrefix + "-heal-success-self", ("target", args.Target),("bible", uid));
             _popupSystem.PopupEntity(selfMessage, args.User, Filter.Entities(args.User));
 
-            SoundSystem.Play(Filter.Pvs(args.Target.Value), "/Audio/Effects/holy.ogg", args.User);
+            SoundSystem.Play(Filter.Pvs(args.Target.Value), component.HealSoundPath, args.User);
             _damageableSystem.TryChangeDamage(args.Target.Value, component.Damage, true);
         }
 
+        private void AddSummonVerb(EntityUid uid, BibleComponent component, GetVerbsEvent<AlternativeVerb> args)
+        {
+            if (!args.CanInteract || !args.CanAccess || !HasComp<BibleUserComponent>(args.User) || component.AlreadySummoned || component.SpecialItemPrototype == string.Empty)
+                return;
+
+            AlternativeVerb verb = new()
+            {
+                Act = () =>
+                {
+                    AttemptSummon(component, args.User);
+                },
+                Text = Loc.GetString("bible-summon-verb"),
+                Priority = 2
+            };
+            args.Verbs.Add(verb);
```

Comment:
Conditions need to be rechecked in the act because it can be issued at any time.